### PR TITLE
1st amap

### PR DESCRIPTION
**Pull request by edgano**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

----------------
[PR](https://github.com/mes5k/amap-align/pull/3) open on Amap repo to fix the compiling issue

### CODE REVIEW

File: recipes/amap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  run:
+    - blast
+
+test:
+  commands:
+    - amap -gui
+
+about:
+  home: "https://web.archive.org/web/20060902044446/http://bio.math.berkeley.edu/amap/"
+  license: "GPL"
```
Comment (bgruening) [inline recipes/amap/meta.yaml]: Do you know which version of GPL? Can you add a license_file?

Thanks!

Comment (edgano) [reply recipes/amap/meta.yaml]: The file is in /license/License.txt on the zip file

should I write something like : ??
```
about:
  home: https://web.archive.org/web/20060902044446/http://bio.math.berkeley.edu/amap/
  license: GPL-2
  license_file: /license/License.txt
  summary: JAMAP is a multiple sequence alignment program based on sequence annealing
  dev_url: https://github.com/mes5k/amap-align
```

Comment (edgano) [reply recipes/amap/meta.yaml]: hey @bgruening I hope it works now. 
But if it's not the best way to add the license just let me know 😄

Comment (dpryan79) [reply recipes/amap/meta.yaml]: It looks good with the most recent changes. I'll merge once it passes testing.

**Comment by dpryan79** [comment]
Thanks @edgano !

**Pull request merged**